instrumentation/logback/logback-appender-1.0/library/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
   id("otel.library-instrumentation")
+  id("org.graalvm.buildtools.native") version "0.9.14"
```

Comment:
Can you add this to the `settings.gradle.kts` `pluginManagement` section, and just refer to the plugin by name here?